masonry_core/src/core/widget.rs

```diff
@@ -410,38 +410,43 @@ pub trait Widget: AsAny + AsDynWidget {
 }
 
 /// See [`Widget::find_widget_at_pos`] for more details.
-pub fn find_widget_at_pos<'c>(
+pub fn find_widget_under_pointer<'c>(
     widget: &WidgetRef<'c, dyn Widget>,
     pos: Point,
 ) -> Option<WidgetRef<'c, dyn Widget>> {
-    if widget.ctx.widget_state.bounding_rect.contains(pos) {
-        let local_pos = widget.ctx().widget_state.window_transform.inverse() * pos;
+    let ctx = widget.ctx();
 
-        if widget.ctx.is_stashed()
-            || Some(false) == widget.ctx.clip_path().map(|clip| clip.contains(local_pos))
-        {
+    if !ctx.bounding_rect().contains(pos) {
+        return None;
+    }
+    if ctx.is_stashed() {
+        return None;
+    }
+
+    let local_pos = ctx.window_transform().inverse() * pos;
+
+    if let Some(clip) = ctx.clip_path() {
+        if !clip.contains(local_pos) {
```

Comment:
I guess we can soon combine these in one `if` (as let chains are just getting stabilized) :)

Reply:
Fingers crossed.